plugins/telemetry-otel/build.gradle

```
@@ -25,6 +27,7 @@ dependencies {
   api "io.opentelemetry:opentelemetry-sdk-common:${versions.opentelemetry}"
   api "io.opentelemetry:opentelemetry-sdk-trace:${versions.opentelemetry}"
   api "io.opentelemetry:opentelemetry-sdk-metrics:${versions.opentelemetry}"
+  api "io.opentelemetry:opentelemetry-sdk-testing:${versions.opentelemetry}"
```

Uh I missed that, sorry - please change it to test scope:

```suggestion
  testImplementation "io.opentelemetry:opentelemetry-sdk-testing:${versions.opentelemetry}"
```